### DISCUSSION TITLE
I compressed the frontend.js...

### DISCUSSION THREAD

**Discussion by c1972** [thorsten/phpMyFAQ#3026] [Ideas]
**PageSpeed-Insights** complained, among other things, **about the file size of frontend.js**.

I compressed the JavaScript code:
Before: 2.1 MB / After: 1.2 MB = 900 KB saved!

I used: https://www.giftofspeed.com/javascript-compressor/

Now PageSpeed-Insights no longer complains about this point.

![gtmetrix-frontendjs](https://github.com/thorsten/phpMyFAQ/assets/112912128/2b769ec0-d6f2-4a8d-9c35-c4390d4ec4e0)

After my edit of the htaccess **Gzip** also finally works. :)

Maybe you can use this to make the page even faster and get a better result in PageSpeed.

Regards, Christian

**Comment by thorsten** [comment]
It's not needed as with the new instructtions in .htaccess the fontend.js file is even smaller now:

<img width="655" alt="Bildschirmfoto 2024-06-26 um 20 04 39" src="https://github.com/thorsten/phpMyFAQ/assets/45284/66c1b9e4-fc07-4479-92bc-2a57692c3f85">

**Comment by c1972** [comment]
Sounds good! :)
